src/django/api/helpers.py

```diff
@@ -20,3 +22,42 @@ def prefix_a_an(value):
     if not CONSONANT_SOUND.match(value) and VOWEL_SOUND.match(value):
         return 'An {}'.format(value)
     return 'A {}'.format(value)
+
+
+def parse_raw_data(data):
```

Comment:
Moved these functions into the `helpers.py` file to allow them to be used in multiple locations